velox/exec/tests/UnnestTest.cpp

```diff
@@ -474,6 +474,47 @@ TEST_P(UnnestTest, batchSize) {
   ASSERT_EQ(expectedNumVectors, stats.at(unnestId).outputVectors);
 }
 
+TEST_P(UnnestTest, basicArrayWithOuter) {
+  auto vector = makeRowVector({
+      makeFlatVector<int64_t>({1, 2, 3}),
+      makeNullableArrayVector<int64_t>({
+        {1, 2, std::nullopt},
+        {},
+        {3}
+      }),
+  });
+
+  createDuckDbTable({vector});
+
+  // is_outer = false
+  auto op1 = PlanBuilder().values({vector}).unnest({"c0"}, {"c1"}, std::nullopt, false /* is_outer */).planNode();
+  auto params1 = makeCursorParameters(op1);
+  auto expected1 = makeRowVector({
```

Comment:
nit: can we have 'op', 'params', 'expected' and reuse them in the other test cases?